src/GameController/GCExtendedGamepadSnapshot.cs

```diff
@@ -49,13 +49,22 @@ public struct GCExtendedGamepadSnapShotDataV100 {
 		// Buttons in the range [0.0, 1.0]
 		public float /* float_t = float */ LeftTrigger;
 		public float /* float_t = float */ RightTrigger;
+		[iOS (12,2), Mac (10,14,4, onlyOn64: true), TV (12,2)]
+		public bool supportsClickableThumbsticks;
+		[iOS (12,2), Mac (10,14,4, onlyOn64: true), TV (12,2)]
+		public bool leftThumbstickButton;
+		[iOS (12,2), Mac (10,14,4, onlyOn64: true), TV (12,2)]
+		public bool rightThumbstickButton;
 
 		// radar: https://trello.com/c/7FoGTORD (GCExtendedGamepadSnapShotDataV100 struct size / alignment not backward compatible)
 		// [TV (12, 1), Mac (10, 14, 1, onlyOn64: true), iOS (12, 1)]
 		// public bool LeftThumbstickButton;
 		// [TV (12, 1), Mac (10, 14, 1, onlyOn64: true), iOS (12, 1)]
 		// public bool RightThumbstickButton;
```

Comment:
So far checking this, the link provided is dead. They also added the "SupportsClickableThumbsticks" which may have helped the functionality of those removed buttons. Where would you recommend checking if these should be here or not?

Reply:
it's not dead - but you need to login (and have access) to trello to see it
Nothing much to see since Apple closed it as a duplicate. The original issue was that the structure was not backward compatible (since fields were added in the middle, not at end).

Reply:
Ah okay, so should I remove the LeftThumbsitckButton and RightThumbsitckButton but leave in the SuportsClickableThumbsticks?

Reply:
no, if the new fields are present (in the headers) at the end of the structure then we must match the memory representation (so we must have the elements in the same order)

Reply:
Sorry I am a little confused. In the xcode11 headers, it seems that it should be in the order of SupportsClickableThumbsticks
LeftThumbstickButton
RightThumbstickButton. 
So you are saying I should include these 3 and put them in this order?

Reply:
I have not looked at the headers... but, since it's a structure, they *must* match (same order, same size...) so that the code from both ObjC and C# can write to the (same) memory location (and make sense of what the other wrote)